examples/pt2/torch_compile_hpu/README.md

```diff
@@ -0,0 +1,142 @@
+
+# TorchServe Inference with torch.compile with HPU backend of Resnet50 model
+
+This guide provides steps on how to optimize a ResNet50 model using `torch.compile` with [HPU backend](https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Getting_Started_with_Inference.html), aiming to enhance inference performance when deployed through TorchServe. `torch.compile` allows for ahead-of-time compilation of PyTorch models.
```

Comment:
`torch.compile` is a JIT compiler. Can you please change this. To help with warmup problem, we use AOT compile @RafLit

Reply:
something like this : `torch.compile` allows for JIT compilation of Python code into optimized kernels with a simple API.

Reply:
The description has been updated according to your suggestion @agunapal